builder/steps/gen-dockerfile/contents/src/detect_setup.ts

```diff
@@ -141,16 +141,19 @@ export async function detectSetup(
     throw new Error(`The file ${appYamlPath} does not exist`);
   }
   const config = yaml.safeLoad(await fsview.read(appYamlPath));
+  if (!config) {
```

Comment:
I'm not sure why `yaml.safeLoad()` can return `undefined` with Node 8.11 but not with older versions... But seems ok.

Reply:
It's because the new release of `@types/js-yaml` changes the return type of `safeLoad` ([See this commit](https://github.com/DefinitelyTyped/DefinitelyTyped/commit/ae9aff4a0c0ccdc224583f0f7f84d3a5b7bc3483#diff-96ad27e3d37ce4d83a5d14df7c47b120)).

The change is independent of node 8.11.0, and has to do with the fact that at the time I made a release for node 8.10.0, the `@types/js-yaml` hadn't changed.